src/main/java/com/vfasad/service/EmailService.java

```diff
@@ -49,7 +45,7 @@ public void notifyManagerOrderCompleted(Order order) {
 
         // send copy to storage
         emailSender.send(
-                mailStorage,
+                System.getenv("MAIL_STORAGE"),
```

Comment:
Please, use property from config here, and make a link to env var in a property file.